File: RecordMusicCunoc/src/main/java/com/adolfosc/modelo/music/Tiempo.java
package com.adolfosc.modelo.music;

import java.io.Serializable;

/**
 *
 * @author hectoradolfo
 */
public class Tiempo implements Serializable {
    
    private int canal;
    private int tiempo;

    public Tiempo(int canal) {
        this.canal = canal;
        this.tiempo = 0;
    }
    
    public int getCanal() {
        return canal;
    }

    public void setCanal(int canal) {
        this.canal = canal;
    }

    public int getTiempo() {
        return tiempo;
    }

    public void setTiempo(int tiempo) {
        this.tiempo = tiempo;
    }
    
}

File: RecordMusicApp/app/src/main/java/com/adolfosc/modelo/music/Tiempo.kt
package com.adolfosc.modelo.music

import java.io.Serializable

class Tiempo: Serializable {

    private var canal = 0
    private var tiempo = 0

    constructor() {
        this.canal = canal
        tiempo = 0
    }

    fun Tiempo(canal: Int) {
        this.canal = canal
        tiempo = 0
    }

    fun getCanal(): Int {
        return canal
    }

    fun setCanal(canal: Int) {
        this.canal = canal
    }

    fun getTiempo(): Int {
        return tiempo
    }

    fun setTiempo(tiempo: Int) {
        this.tiempo = tiempo
    }
}